dkobo/koboform/api.py

```diff
@@ -26,15 +40,26 @@ def get_queryset(self):
         return queryset.order_by('-date_modified')
 
     def create(self, request):
-        user = self.request.user
-        if user.is_anonymous():
-            raise PermissionDenied
         contents = request.DATA
         tags = contents.get('tags', [])
         if 'tags' in contents:
             del contents['tags']
 
-        survey_draft = request.user.survey_drafts.create(**contents)
+        myw_kobo_user_cookie = self.request.COOKIES.get('myw_kobo_user')
```

Comment:
Isn't this exact same code repeated on lines 28-34? Should be abstracted, kept DRY